flink-connectors/flink-connector-cassandra/src/main/java/org/apache/flink/streaming/connectors/cassandra/NoOpCassandraFailureHandler.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.streaming.connectors.cassandra;
+
+import org.apache.flink.annotation.Internal;
+
+import java.io.IOException;
+
+/**
+ * A {@link CassandraFailureHandler} that simply fails the sink on any failures.
+ */
+@Internal
+public class NoOpCassandraFailureHandler implements CassandraFailureHandler {
```

Comment:
Because this handler throws an exception, I think the name "NoOp" may not be accurate, maybe named `FailingCassandraFailureHandler` (here, I just refer to `FailingCheckpointExceptionHandler`) or some other name is more appropriate .

Reply:
There is a `NoOpFailureHandler` used in ElasticsearchSink which throws the exception as well. I referred to that class while naming the failureHandler in CassandraSink.
Should I rename the class?

Reply:
Oh, I didn't go to see the related implementation of `ElasticsearchSink `. This is just a suggestion, because when I first saw the name, I thought it was an empty method, didn't do anything because you called it "NoOp", and in fact you throw an exception, it will cause the sink task to fail if the outer layer does not handle it. Maybe you can listen to other people's opinions and see if it is a better choice to keep up with the implementation of `ElasticsearchSink `.

Reply:
Hm, I read NoOp as not doing anything at all and allowing all exceptions to pass through :)
I can wait for other reviewers to suggest better notation.
Thanks!

Reply:
@tillrohrmann any suggestions?

Reply:
@yanghua Hi, should I just commit with the other indentation fix, or should I wait for consensus on naming of NoOpFailureHandler?

Reply:
@wittyameta You don't have to modify this naming. This is just my personal thought and does not represent the final reviewer's opinion. In addition, PR review may not be so fast, please be patient.